### ISSUE TITLE
Fix evaluation the transparent model

### ISSUE THREAD

**Issue by YusukeHirao** [markuplint/markuplint#617]
- Markuplint version: `3.0.0-rc.4`
- Parser lang: HTML
- Node.js version: `16`
- OS: macOS

## Describe the bug

It warns when a transparent model element is a component root.

## Code Example

```html
<noscript>
  <style></style>
</noscript>
```

## Steps To Reproduce

1. Write code
2. Run Markuplint

## The current behavior

> error: The "style" element is not allowed in the "noscript" element through the transparent model in this context (permitted-contents)

## The expected behavior

No error
